account/manager.go

```diff
@@ -480,26 +468,23 @@ func (m *Manager) resumeAccount(ctx context.Context, account *Account, // nolint
 		}
 
 		if createTx {
-			// We need a static sat/vByte value for the fee in
-			// SendOutputs, so we use the stored targetConf of the
-			// account to estimate it.
-			feeSatPerKw, err := m.cfg.Wallet.EstimateFee(
-				ctx, int32(fundingConfTarget),
-			)
-			if err != nil {
-				return err
+			acctKey := account.TraderKey.PubKey.SerializeCompressed()
+
+			if feeRate == 0 {
+				return fmt.Errorf("unable to create transaction for "+
+					"account with trader key %x, feeRate should be greater "+
```

Comment:
nit: wrap at 80 characters. You might need to update your editor's config to use 8 spaces for a tab character instead of just 4 or 2 to match everyone else's view of what 80 characters width is :)

Reply:
I wonder if we can use a lint step for this, like `lll` or something

Reply:
The problem is, some lines we can't really break nicely (for example the config with the tagged struct values). So we'd get a lot of false positives. And adding ` // nolint:lll` to each of the long lines just makes them even longer :sweat_smile: